apps/benchmarks/src/App.tsx

```diff
@@ -25,78 +28,155 @@ function App() {
+        })
+
+        timeValues.push(time4)
+
+        setV4Times(timeValues.slice())
+
+        const [, time5] = await run(() => {
+            group.updateMember(0, 2n)
+        })
+
+        timeValues.push(time5)
+
+        setV4Times(timeValues.slice())
+    }, [v4GroupMembers])
 
     return (
         <Flex flexDir="column" flex="1">
             <Navbar />
             <Flex flex="1" align="center" justify="center">
-                <Flex flexDir="column" gap={4} width="400px">
-                    <TreeDepth value={treeDepth} onChange={setTreeDepth} />
-                    <GroupMembers value={groupMembers} onChange={setGroupMembers} max={2 ** treeDepth} />
-
-                    <Button onClick={() => runFunctions()} size="sm" my="3">
-                        Run functions
-                    </Button>
-
-                    <List spacing={3}>
-                        {functions.map((f, i) => (
-                            <ListItem key={f}>
-                                <Flex justify="space-between">
-                                    <Box>
-                                        {times[i] && <ListIcon as={MdCheckCircle} color="green.500" />}
-                                        <b>{f}</b>
-                                    </Box>
-                                    <Text>{times[i] ? times[i] : 0} ms</Text>
-                                </Flex>
-                            </ListItem>
-                        ))}
-                    </List>
+                <Flex gap={10}>
+                    <Flex flexDir="column" gap={4} width="300px">
+                        <Heading as="h3" size="lg">
+                            Semaphore v3
+                        </Heading>
+                        <TreeDepth value={v3treeDepth} onChange={setV3TreeDepth} />
+                        <GroupMembers value={v3GroupMembers} onChange={setV3GroupMembers} max={2 ** v3treeDepth} />
+
+                        <Button onClick={() => runV3Functions()} size="sm" my="3">
+                            Run functions
+                        </Button>
+
+                        <List spacing={3}>
+                            {functions.map((f, i) => (
+                                <ListItem key={f}>
+                                    <Flex justify="space-between">
+                                        <Box>
+                                            {v3Times[i] && <ListIcon as={MdCheckCircle} color="green.500" />}
+                                            <b>{f}</b>
+                                        </Box>
+                                        <Text>{v3Times[i] ? v3Times[i] : 0} ms</Text>
+                                    </Flex>
+                                </ListItem>
+                            ))}
+                        </List>
+                    </Flex>
+                    <Flex flexDir="column" gap={4} width="300px">
+                        <Heading as="h3" size="lg">
```

Comment:
```suggestion
                        <Heading as="h3" size="lg"  mb="12">
```

What about adding a margin there so that it's easier to compare the functions side by side?